docs/berkeley-upgrade/archive-migration-installation.mdx

```diff
@@ -115,9 +115,15 @@ apt-get install --allow-downgrades -y "mina-archive-migration=$VERSION"
 To get the Docker image:
 
 ```
-docker pull gcr.io/o1labs-192920/mina-archive-migration:3.0.0-rc0
+docker pull gcr.io/o1labs-192920/mina-archive-migration:3.0.0-rc0-{CODENAME}
```

Comment:
Let's update this tag

Reply:
What you mean? should we stick to 3.0.0-rc0 ? i think we need to differentiate between codenames

Reply:
I think @amc-ie suggested 3.0.0-release-migration or similar